components/pipedream/actions/create-subscription/create-subscription.mjs

```diff
@@ -0,0 +1,34 @@
+import pipedream from "../../pipedream.app.mjs";
+
+export default {
+  key: "pipedream-create-subscription",
+  name: "Creates a Subscription",
+  description: "Create a Subscription. [See Doc](https://pipedream.com/docs/api/rest/#subscriptions)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    pipedream,
+    emitterId: {
+      propDefinition: [
+        pipedream,
+        "emitterId",
+      ],
+    },
+    eventName: {
+      type: "string",
+      label: "Event name",
+      description: "Event name",
+    },
+    listenerId: {
+      type: "string",
+      label: "Listener ID",
+      description: "Listener ID",
+    },
+  },
+  async run({ $ }) {
+    const response = await this.pipedream
+      .subscribe(this.emitterId, this.listenerId, this.eventName);
+    response?.data?.length > 0 &&  $.export("$summary", "Successfully created Subscription");
```

Comment:
What happens if `response.data.length === 0 || response.data.length === undefined`?
Should the user be notified if the subscription wasn't created as expected?

Reply:
I tried following the response data structure which returns {data:Object[]} on success